index.html

```diff
@@ -16,13 +16,13 @@
 </header>
     <main>
         <h1>Welcome to Jordan Poulin's CSD120 Website!</h1>
-            <p>
-             I'm currently a first year Computer Programmer student at Sault College in Sault Ste. Marie, Ontario. The majority of <br>
-             my IT experience is in hardware. I'm an avid gamer and enjoy building PCs and figuring out how computer components work. <br>
-             My goal in taking this course is to expand my knowledge out of hardware and more into software development and other forms <br>
-             of programming, such as web development, as well as pursue a career in this field.                                       <br>
-                                                                                                                                 <br>
-             In my spare time I like to relax and spend time with my family. I play a good amount of games on my PC, but I try not to <br>
+          <p>
```

Comment:
Use multiple `<p>` elements to markup separate paragraphs rather that adding `<br>` tags